cloud/pkg/cloudhub/handler/eventhandler.go

```diff
@@ -94,6 +97,13 @@ func (eh *EventHandle) HandleServer(container *mux.MessageContainer, writer mux.
 		return
 	}
 
+	// handle the reponse from edge
+	matched, _ := regexp.MatchString(VolumeRegExp, container.Message.GetResource())
```

Comment:
You can use `func (re *Regexp) MatchString(s string) bool` instead of `func MatchString(pattern string, s string) (matched bool, err error)` to get higher runtime performance.

Reply:
Done. Thanks